### ISSUE TITLE
Add WIB2 to WIBEthernet conversion script

### ISSUE THREAD

**Issue by adam-abed-abud** [DUNE-DAQ/rawdatautils#36]
In the current repository it seems that there is only a tool to convert ProtoWIB data to either DUNEWIB (WIB2) or WIBEth. It would be useful to have a WIB2 to WIBEth. A possible use-case would be to convert some of the raw recordings into WIBEthernet for sw validation (e.g. SWTPG)

**Comment by wesketchum** [comment]
I don't think this is needed anymore. Closing the issue until there's another request.
